TTHAnalysis/macros/prepareEventVariablesFriendTree.py

```diff
@@ -263,12 +99,16 @@ def analyze(self,event):
     exit()
 
 if "{P}" in args[1]: args[1] = args[1].replace("{P}",args[0])
-if len(args) != 2 or not os.path.isdir(args[0]):
+if len(args) != 2:
     print "Usage: program <TREE_DIR> <OUT>"
     exit()
-if not os.path.isdir(args[1]): 
-    os.system("mkdir -p "+args[1])
-    if not os.path.isdir(args[1]): 
+if not os.path.isdir(args[0]):
+    print "Error. Input directory {input} does not exist".format(input=args[0])
+    exit()
+if not os.path.isdir(args[1]):
+    tempOut = args[1].replace('/pool/ciencias/','/pool/cienciasrw/')
```

Comment:
@gpetruc @peruzzim  It is needed, unfortunately, as I pointed out in the previous answer.

We did put the hack also in another couple places, it should be enough (we produced friend trees correctly with the current configuration): Do you have any specific portion of code in mind?

Pietro